src/gtest/test_forkmanager.cpp

```diff
@@ -303,7 +432,7 @@ TEST(ForkManager, SidechainVersionForkTestnet) {
     SelectParams(CBaseChainParams::TESTNET);
 
     // TODO: set proper fork height value.
```

Comment:
This comment can be removed.
```suggestion
```

Reply:
Addressed in https://github.com/HorizenOfficial/zen/commit/6f1ffaeccf187e2903bfa608c54c5138fe7d834c